StratisCore.UI/src/app/setup/setup.component.ts

```diff
@@ -1,14 +1,22 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import {Location} from '@angular/common';
 import { Router } from '@angular/router';
+import { GlobalService } from '@shared/services/global.service';
 
 @Component({
   selector: 'setup-component',
   templateUrl: './setup.component.html',
   styleUrls: ['./setup.component.css'],
 })
-export class SetupComponent {
-  constructor(private router: Router, private location: Location) {}
+export class SetupComponent implements OnInit {
+  constructor(private router: Router, private globalService: GlobalService) {}
+
+  public sidechainEnabled: Boolean;
```

Comment:
I think the best practive is not to use Boolean (the one with capital B), because
```
var falseObj = new Boolean(false);
if (falseObj) {
    console.log('our intent may not be to get here');
}
```